packages/usa-validation/src/test/template.html

```diff
@@ -1,15 +1,24 @@
 <div>
   <ul class="usa-checklist" id="validation_list">
-    <li class="usa-checklist__item" data-validator="length">Be at least 8 characters</li>
-    <li class="usa-checklist__item" data-validator="uppercase">Have at least 1 uppercase character</li>
-    <li class="usa-checklist__item" data-validator="numerical">Have at least 1 numerical character</li>
+    <li class="usa-checklist__item" data-validator="length">
+      Be at least 8 characters
+    </li>
+    <li class="usa-checklist__item" data-validator="uppercase">
+      Have at least 1 uppercase character
+    </li>
+    <li class="usa-checklist__item" data-validator="numerical">
+      Have at least 1 numerical character
+    </li>
   </ul>
   <input
     class="usa-input"
-    id="password" name="password" type="password"
+    id="password"
+    name="password"
+    type="password"
     aria-describedby="validation_list"
     data-validate-length=".{8,}"
     data-validate-uppercase="[A-Z]"
     data-validate-numerical="\d"
-    data-validation-element="validation_list">
+    data-validation-element="#validation_list"
```

Comment:
@mejiaj Is the `#` in the data attribute meaningful? Otherwise the markup is exactly the same?

Reply:
```suggestion
    data-validation-element="validation_list"
```